aten/src/ATen/FunctionalizeFallbackKernel.cpp

```diff
@@ -181,14 +181,26 @@ static at::Tensor lift_functionalize(const at::Tensor & self) {
 }
 
 static at::Tensor lift_fresh_functionalize(const at::Tensor & self) {
-  TORCH_INTERNAL_ASSERT(!at::functionalization::impl::isFunctionalTensor(self));
+  // See Note [Exporting and compiling a graph with lift_fresh_copy]
+  if (at::functionalization::impl::isFunctionalTensor(self)) {
+    return self;
```

Comment:
You CANNOT return self here as you're below autograd.
```suggestion
    return self.view_as(self);
```

Reply:
And I thought i'd finally drilled that into my head